config/routes.rb

```diff
@@ -4,6 +4,7 @@
     get "/help", to: "static_pages#help"
     get "/contact", to: "static_pages#contact"
     get "/signup", to: "users#new"
-    resources :users, only: [:new]
+    post "/signup", to: "users#create"
+    resources :users, only: %i(new create show)
```

Comment:
```suggestion
    resources :users, only: %i(show)
```
2 router này thừa, mình đã có trong phần /signup r